modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -422,6 +422,9 @@ public String getTypeDeclaration(Property p) {
         if (p instanceof ArrayProperty) {
             ArrayProperty ap = (ArrayProperty) p;
             Property inner = ap.getItems();
+            if (inner == null) {
+              return null;
```

Comment:
Instead of returning `null`, what about defaulting to `string` and log (LOGGER.WARN) a warning message so that the user is aware of the missing inner type)?